app/requirements.txt

```diff
@@ -1,5 +1,5 @@
-ddtrace==0.11.0
+ddtrace==0.27.1
 elasticsearch>=5.0.5,<6.0.0
 redis==2.10.6
 falcon==1.4.1
-uwsgi==2.0.15
+uwsgi==2.0.18
```

Comment:
oh, we're only like 2 years behind, which `pay` and `mes` have also inherited now: https://github.com/unbit/uwsgi/compare/2.0.15...2.0.18